superset-frontend/src/pages/Tags/index.tsx

```diff
@@ -59,6 +59,17 @@ function TagList(props: TagListProps) {
   const { addDangerToast, addSuccessToast, user } = props;
   const { userId } = user;
 
+  const initialFilters = useMemo(
+    () => [
+      {
+        id: 'type',
+        operator: 'custom_tag',
```

Comment:
Is there ever a case where the client would need to use this api to return system-generated tags? I'm wondering if this filter should just be applied by default server-side.

Reply:
@eschutho I thought about that as well (wasn't sure if we ever wanted to expose those system-tags). I think the biggest advantage with it being a filter is the flexibility. 

Could be useful for example in case an admin wants to retrieve all assets favorited by user X, or owned by user Y.